impit-python/src/response.rs

```diff
@@ -32,6 +32,16 @@ pub(crate) struct ImpitPyResponse {
     // elapsed: Duration,
 }
 
+#[pymethods]
+impl ImpitPyResponse {
+    fn __repr__(&self) -> PyResult<String> {
+        Ok(format!(
+            "<Response [{} {}]>",
+            self.status_code, self.reason_phrase
+        ))
+    }
```

Comment:
```suggestion
    fn __repr__(&self) -> String {
        format!(
            "<Response [{} {}]>",
            self.status_code, self.reason_phrase
        )
    }
```

We don't need to return `Result`, as this code cannot produce an error. There is no harm in doing it (as this will probably be called exclusively from Python), but if we would want to call `__repr__` from Rust code, we'd have to `unwrap()` the return value (needs more code to be used).